src/report.rs

```diff
@@ -444,7 +444,7 @@ impl From<RawData> for Report {
                         "💯 Batch finalized at block #{} 
                     (<a href=\"https://{}.subscan.io/extrinsic/{:?}\">{}</a>) ✨",
                         batch.block_number,
-                        data.network.name.to_lowercase().trim().replace(" ", ""),
+                        data.network.name.to_lowercase().trim().replace(" ", "-"),
```

Comment:
Similar question as well. I see the string replacement but what is the expected result ?